ewhunting.py

```diff
@@ -0,0 +1,1553 @@
+
+                        # Player was killed. Remove its id from enemies with defender ai.
+                        enemy_data.id_target = ""
+                        target_data.id_killer = enemy_data.id_enemy
+                        target_data.die(cause=ewcfg.cause_enemy_killing)
+                        target_data.change_slimes(n=-slimes_dropped / 10, source=ewcfg.source_ghostification)
+
+                        kill_descriptor = "beaten to death"
+                        if used_attacktype != ewcfg.enemy_attacktype_unarmed:
+                            response = used_attacktype.str_damage.format(
+                                name_enemy=enemy_data.display_name,
+                                name_target=member.display_name,
+                                hitzone=randombodypart,
+                                strikes=strikes
+                            )
+                            kill_descriptor = used_attacktype.str_killdescriptor
+                            if crit:
+                                response += " {}".format(used_attacktype.str_crit.format(
+                                    name_enemy=enemy_data.display_name,
+                                    name_target=member.display_name
+                                ))
+
+                            response += "\n\n{}".format(used_attacktype.str_kill.format(
+                                name_enemy=enemy_data.display_name,
+                                name_target=member.display_name,
+                                emote_skull=ewcfg.emote_slimeskull
+                            ))
+                            target_data.trauma = used_attacktype.id_type
+
+                        else:
+                            response = "{name_target} is hit!!\n\n{name_target} has died.".format(
+                                name_target=member.display_name)
+
+                            target_data.trauma = ""
+
+                        if target_slimeoid.life_state == ewcfg.slimeoid_state_active:
+                            brain = ewcfg.brain_map.get(target_slimeoid.ai)
+                            response += "\n\n" + brain.str_death.format(slimeoid_name=target_slimeoid.name)
+
+                        deathreport = "You were {} by {}. {}".format(kill_descriptor, enemy_data.display_name,
+                                                                     ewcfg.emote_slimeskull)
+                        deathreport = "{} ".format(ewcfg.emote_slimeskull) + ewutils.formatMessage(member, deathreport)
+
+                        target_data.persist()
+                        enemy_data.persist()
+                        resp_cont.add_channel_response(ewcfg.channel_sewers, deathreport)
+                        resp_cont.add_channel_response(ch_name, response)
+                        if ewcfg.mutation_id_spontaneouscombustion in target_mutations:
+                            import ewwep
```

Comment:
this seems like extremely poor form. if you're having issues with circular imports, I'd rather you move explode() to ewutils or something